src/App.test.js

```diff
@@ -0,0 +1,8 @@
+// import { render, screen } from '@testing-library/react';
+// import App from './App';
+
+// test('renders learn react link', () => {
+//   render(<App />);
+//   const linkElement = screen.getByText(/learn react/i);
+//   expect(linkElement).toBeInTheDocument();
+// });
```

Comment:
- [ ] Kindly delete unwanted or commented code to make your project more professional.
- [ ] Please make sure that the tests run and pass successfully, this is what I got when I ran the tests.

![image](https://github.com/sheezu900/React-Capstone-Project/assets/62290545/f32af6b6-5f92-4f19-9d62-b93f03f0fc64)